### ISSUE TITLE
Sign In Invalid Arguments

### ISSUE THREAD

**Issue by enso123456** [StephenGrider/AdvancedReduxCode#21]
Server Post Sign In Invalid Arguments it will occur in the bcrypt.comparePassword method.

**Comment by afisherdev** [comment]
Yeah I am having the same issue. The problem is in the user.js file. The  this.password argument console.logs as undefined. I downloaded the source, just to make sure that it wasn't an error that I created, its the same in the source material.
